staticServer/router.go

```diff
@@ -0,0 +1,50 @@
+package staticServer
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/gorilla/mux"
+)
+
+// NewRouter сreates and returns gorilla router
+func NewRouter() *mux.Router {
+	r := mux.NewRouter()
+	r.Handle("/", http.FileServer(http.Dir("../../staticServer/static")))
+	r.PathPrefix("/static/").Handler(http.StripPrefix("/static/", http.FileServer(http.Dir("../../staticServer/static"))))
+	r.Path("/content").HandlerFunc(fetch("fridgeContent")).Methods("GET")
+	r.Path("/recipes").HandlerFunc(fetch("allRecipes")).Methods("GET")
```

Comment:
Do not hardcode these values that you pass to `fetch`. They could be changed.

And please think about using this approach with single `fetch` function. It would be hard to use it further when you have many different endpoints.

Reply:
Should I also add them to .yaml? :cold_sweat: everything can change... Maybe we'll rewrite it entirely to pass data to clients with protobufs... maybe it's not that much to change that endpoint manually? :sweat_smile:
About `fetch`... that type of functions does exactly the same thing... It's just transits json data. And every get request to server will behave like this. Is it better to write it twice? :sweat_smile: